ReactiveCocoa/Swift/Signal.swift

```diff
@@ -979,29 +961,28 @@ extension SignalType {
 
 	/// Waits until `self` completes and then forwards the final `count` values
 	/// on the returned signal.
-	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
-	public func takeLast(count: Int) -> Signal<Value, Error> {
+	public func takeLast(_ count: Int = 1) -> Signal<Value, Error> {
```

Comment:
If we rename all the other tags as above, it feels like this should be `take(last:)`. I know that's out of line with the stdlib though. :confused: